draft-ietf-spice-sd-cwt.md

```diff
@@ -196,122 +279,78 @@ sd-protected = {
    * key => any
 }
 
-unprotected-presentation = {
-   &(sd_kbt: TBD2) ^ => bstr .cbor kbt-cwt,
-   ? &(sd_claims: TBD1) ^ => bstr .cbor [ + bstr .cbor salted ],
+unprotected-issued = {
+   &(sd_claims: TBD1) ^ => bstr .cbor salted-array,
    * key => any
 }
 
 sd-payload = {
     ; standard claims
       &(iss: 1) ^ => tstr, ; "https://issuer.example"
       &(sub: 2) ^ => tstr, ; "https://device.example"
-      &(aud: 3) ^ => tstr, ; "https://verifier.example"
+    ? &(aud: 3) ^ => tstr, ; "https://verifier.example/app"
     ? &(exp: 4) ^ => int,  ; 1883000000
     ? &(nbf: 5) ^ => int,  ; 1683000000
       &(iat: 6) ^ => int,  ; 1683000000
       &(cnf: 8) ^ => { * key => any }, ; key confirmation
     ? &(cnonce: 39) ^ => bstr,
     ;
     ; sd-cwt new claims
+      # TODO: should sd_hash be included in issued?
+      &(sd_hash: TBD3) ^ => bstr,
       &(sd_alg: TBD4) ^ => int,            ; -16 for sha-256
-    ? &(redacted_values: TBD5) ^ => [ * bstr ],
+    ? &(redacted_keys: TBD5) ^ => [ * bstr ],
```

Comment:
I would change all instances of "redacted_keys" to "redacted_values".  It is unfortunate that CBOR chose to use the term "key" for the labels of objects, as it inevitably causes readers confusion between cryptographic keys and object keys.

Please let's reserve the term "key" for cryptographic keys.

Reply:
> I would change all instances of "redacted_keys" to "redacted_values". It is unfortunate that CBOR chose to use the term "key" for the labels of objects, as it inevitably causes readers confusion between cryptographic keys and object keys.
> 
> Please let's reserve the term "key" for cryptographic keys.

"values" doesn't convey the difference between array items map elements. keys isn't really correct either.  How do folks feel about `redacted_elements`?

Reply:
I proposed `redacted_map_claims` in Issue #15 . Let's please merge this PR while these other issues get discussed.

Reply:
Marking resolved. Captured in Issue #15